README.md

```diff
@@ -117,10 +115,16 @@ shell with:
 If you're running inside of an msys shell, however, you can run:
 
 ```sh
-$ ./configure --build=x86_64-pc-windows-msvc
-$ make && make install
+$ ./x.py build --build=x86_64-pc-windows-msvc
+$ ./x.py dist --build=x86_64-pc-windows-msvc --install
```

Comment:
Is this a configuration that is recommended for new users on Windows? If not, it's probably fine removing it. (I don't do any dev on Windows, so I don't know the way to go here).